### PR TITLE
Add additional logging

### LINKED ISSUE #184: Logging of robot information from ISAR
**Describe the improvement you would like to see**
With the changes in the robot interface introduced in v1.3.0 the functionality for logging robot specific information was removed from the interface. For debugging purposes it might be desirable to log what is going on in the robot specific implementation. An improvement would be to extract robot specific information that can make the debugging process easier and log them on regular intervals from ISAR.

**How will this change existing functionality?**
This might lead to breaking changes in the robot interface. 

**How will this improvement affect the current Threat Model?**
Logging of the wrong type of information (GDPR, etc) might lead to changes in the Threat Model.

### PR DESCRIPTION
**Pull request by aestene**
Closes #184

### CODE REVIEW

**Comment by aestene** [comment]
Example of log from running `isar_robot` mission
image

File: src/isar/apis/schedule/start_mission.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def post(
             description="ID-number for predefined mission",
         ),
     ):
+        self.logger.info(f"Received request to start new mission")
```
Comment (Christdej) [inline src/isar/apis/schedule/start_mission.py:38]:
```suggestion
        self.logger.info("Received request to start new mission")
```

**Review by Christdej — CHANGES_REQUESTED** [review]
Just a small comment, else looks good!

**Comment by vetlek** [comment]
Do we need to have the whole UUID of the Task in the overview of the tasks? Should be enough to use the first part to uniquely identify the task.

File: src/isar/state_machine/states/send.py [hunk 2]
```diff
@@ -95,10 +97,14 @@ def _run(self):
...
                 next_state = States.Monitor
+                self.logger.info(
+                    f"Successfully scheduled {type(self.state_machine.current_task).__name__} "
+                    f"task: {self.state_machine.current_task.id}"
+                )
                 break
             else:
                 self.send_failure_counter += 1
-                self.logger.info("sending failed #: " + str(self.send_failure_counter))
+                self.logger.info("Sending failed #: " + str(self.send_failure_counter))
```
Comment (vetlek) [inline src/isar/state_machine/states/send.py:107]: Use f-string instead

File: src/isar/apis/schedule/start_mission.py (hunk 1 above)
Comment (aestene) [reply src/isar/apis/schedule/start_mission.py]: Done

**Comment by aestene** [comment]
> Do we need to have the whole UUID of the Task in the overview of the tasks? Should be enough to use the first part to uniquely identify the task.

Fixed.

File: src/isar/state_machine/states/send.py (hunk 2 above)
Comment (aestene) [reply src/isar/state_machine/states/send.py]: Done

**Review by vetlek — APPROVED** [review]
LGTM

**Pull request merged**